### ISSUE TITLE
regarding privacy

### ISSUE THREAD

**Issue by claustromaniac** [da2x/amp2html#7]
I suggest you to state in the description on AMO that this extension won't prevent users from connecting to Google servers (since it needs the client to fetch the AMP document in the first place to work).

Making that clear will hopefully prevent users from assuming that this extension is meant to protect their privacy, which will most likely save you some bad reviews in the future :)

I leave this link here for anyone curious:
https://github.com/ghacksuserjs/ghacks-user.js/issues/424

**Comment by da2x** [comment]
The review is accurate.

There are no claims of nor expectations for increased privacy when using this extension. It really does what it says on the tin: redirect from AMP to HTML. This inplies you have to open the AMP page to be redirected away from it. The extension is meant as a consumer opt-out of Google's attempt at monopolizing content delivery through their AMP Project and content delivery proxy.

I don't believe any description shorter than ten pages of text can clarify how this works beyond what the name of the extension already says that it does.

**Comment by claustromaniac** [comment]
> The review is accurate.

I'm not denying that.

> It really does what it says on the tin: redirect from AMP to HTML. This inplies you have to open the AMP page to be redirected away from it.

For people that don't understand how these technologies work, it doesn't necessarily imply that. I've seen at least two people assume that it redirects to canonical URLs **before** fetching the AMP document (which **we** know can't be done reliably).

> I don't believe any description shorter than ten pages of text can clarify how this works beyond what the name of the extension already says that it does.

I did not suggest you to elaborate on how this extension works. I suggested you to make an _explicit statement_ that this extension is not meant to protect anyone's privacy, because some people already get that false impression from this extension, because they simply don't know how AMP works.

It was only a suggestion, regardless.
